spec.go

```diff
@@ -78,21 +86,19 @@ func (c *SpecController) Show(ctx *app.ShowSpecContext) error {
 	return ctx.OK(b)
 }
 
-// clone does a shallow clone of the repo in the given directory and returns the "go1" or - if there
-// is no go1 branch - the "master" branch SHA.
-func clone(repo, tmpDir string) (string, error) {
-	var branch string
-	clone := func() error {
+// clone does a shallow clone of the repo in the given directory and return the SHA
+// If there is no branch specified, the "master" branch is used.
+func clone(repo, tmpDir, branch string) (string, error) {
+	shallowClone := func() error {
 		gitCmd := exec.Command("git", "clone", "--depth=1", "--single-branch", "--branch", branch, repo)
 		gitCmd.Dir = tmpDir
 		return gitCmd.Run()
 	}
-	branch = "go1"
```

Comment:
The algorithm should still try `go1` first then `master` if no branch is specified (i.e. branch is `""`).